File: Security/src/AspDotNet4/CloudFoundrySingleSignon/Startup.cs
using CloudFoundrySingleSignon.App_Start;
using Microsoft.Owin;
using Owin;

[assembly: OwinStartup(typeof(CloudFoundrySingleSignon.Startup))]

namespace CloudFoundrySingleSignon
{
    public partial class Startup
    {
        public void Configuration(IAppBuilder app)
        {
            ApplicationConfig.RegisterConfig("development");
            ConfigureAuth(app);
        }
    }
}

File: Security/src/AspDotNet4/CloudFoundrySingleSignon-VB/Startup.vb
Imports Microsoft.Owin
Imports Owin

<Assembly: OwinStartup(GetType(CloudFoundrySingleSignon.Startup))>
Namespace CloudFoundrySingleSignon

    Public Class Startup
        Public Sub Configuration(ByVal app As IAppBuilder)
            ApplicationConfig.RegisterConfig("development")
            Security.ConfigureAuth(app)
        End Sub

    End Class
End Namespace
